.github/CODEOWNERS

```
@@ -0,0 +1,3 @@
+/pages/aws/* @opennextjs/aws
+/pages/cloudflare/* @opennextjs/cloudflare
```

@james-elicx is doing a lot of work with us, could you please add him
```suggestion
/pages/cloudflare/* @opennextjs/cloudflare @james-elicx 
```